phovea.js

```diff
@@ -149,7 +149,6 @@ module.exports = function(registry) {
   }, {
     name: 'BoxPlot',
     icon: function() { return System.import('./src/assets/box_icon.png'); },
-    scaling: 'aspect',
```

Comment:
The magic constants on lines 75-76 in boxplot.ts :) I am rewriting it a bit to incorporate your notes though.